docs/introduction/blockchain-representation.rst

```diff
@@ -2,51 +2,51 @@
 Internal representation of a Blockchain
 ***************************************
 
-Being a distributed architecture, the sidechain software is meant to be delivered as a software application that will be compiled/installed by potentially many different independent, connected computers. In blockchain jargon, these computers are called “Nodes,” and the term “node” is also generally used to name the blockchain software itself.
-So, the output of the Sidechain SDK, when customized by a developer, is a “Node” that implements core functionalities, and the added logic.
+Being a distributed architecture, the sidechain software is delivered as an application that will be compiled/installed on potentially many different independent, networked computers. In blockchain jargon, these computers are called “nodes,” and the term “node” is also generally used to name the blockchain software itself.
+So, the output of the Sidechain SDK, when customized by a developer, is a “node” that implements core functionalities and the added sidechain logic.
 
-A Node consists of 4 main elements: “**History**,” “**State**,” “**Wallet**,” and “**Memory pool**.” Before we get to know these 4 elements we need to know what a “box” is.
+A node consists of 4 main elements: “**History**,” “**State**,” “**Wallet**,” and “**Memory pool**.” Before we get to know these 4 elements we need to know what a “box” is.
 
 Concept of a BOX
 ****************
 
 A box generalizes the concept of Bitcoin’s UTXOs.
-A box is a cryptographic object that can be created with some secret keys. This box can be open \ (spent) by the owner of those secret keys. Once opened by the owner of the secret keys the box may not be opened again.
+A box is a cryptographic object that can be created with some (one private key or multiple private keys?) secret keys . This box can be opened (spent) by the owner(owners ?) of those secret keys. Once opened by the owner (owners ?) of the secret keys the box may not be opened again.
```

Comment:
@zennermac 

A box is a cryptographic object that can be created with secret keys . This box can be opened (spent) by the owner of those secret keys. The box may not be opened again after it is opened by the owner of the secret keys .